### PR TITLE
FEATURE: get_plot_data()

### PR DESCRIPTION
**Pull request by gkorompi**

### CODE REVIEW

File: src/pyedb/dotnet/edb_core/nets.py [hunk 1]
```diff
@@ -472,7 +472,7 @@ def _get_points_for_plot(self, my_net_points):
     @pyedb_function_handler()
     def get_plot_data(
         self,
-        nets,
+        nets=None,
```
Comment (SMoraisAnsys) [inline src/pyedb/dotnet/edb_core/nets.py:475]: Could you update the description to:
- add `optional` to `nets` types (this should already have been performed)
- specify that, by default, all nets will be considered

**Review by SMoraisAnsys — CHANGES_REQUESTED** [review]
@svandenb-dev @ring630 Can one of you confirm the changes as it will modify the default behavior of the method.

File: src/pyedb/dotnet/edb_core/nets.py [hunk 2]
```diff
@@ -483,8 +483,8 @@ def get_plot_data(
 
         Parameters
         ----------
-        nets : str, list
-            Name of the net or list of nets to plot. If `None` all nets will be plotted.
+        nets : str, list, optional
+            Name of the net or list of nets to plot. If `None` all nets will be plotted. (default)
```
Comment (SMoraisAnsys) [inline src/pyedb/dotnet/edb_core/nets.py:487]:
```suggestion
            Name of the net or list of nets to plot. If `None` (default value) all nets will be plotted.
```

**Comment by ring630** [comment]
> @svandenb-dev @ring630 Can one of you confirm the changes as it will modify the default behavior of the method.

@SMoraisAnsys it is a bug. When nets==None, all nets are plotted. However, nets is a mandatory input, which makes no sense.

**Review by ring630 — APPROVED** [review]
LGTM

**Review by SMoraisAnsys — APPROVED** [review]

**Pull request merged**